app/models/peoplefinder/concerns/completion.rb

```diff
@@ -21,7 +15,7 @@ def completion_score_fields
       lambda {
         where(%{
           COALESCE(image,'') = ''
-          OR COALESCE(location,'') = ''
+          OR COALESCE(location_in_building,'') = ''
```

Comment:
Didn't we say that all the location fields have to be filled for the profile to be complete? I'll ask Jon.